sdk/spring-sdk/src/main/java/kalix/springsdk/annotations/Query.java

```diff
@@ -34,4 +34,6 @@
    * for composition.
    */
   String value();
+
+  boolean streamUpdates() default false;
```

Comment:
Worth adding some javadocs maybe?

Reply:
Absolutely

Reply:
Picked text from the protobuf with some adaptation. Also sent a PR to proxy to improve text there (see https://github.com/lightbend/kalix-proxy/pull/1693).

Looking for feedback on text clarity.

Reply:
looks good to me